src/assets/toolkit/styles/components/message.css

```diff
@@ -0,0 +1,34 @@
+/** @define Message */
+
+:root {
+  --Message-padding: var(--space-xs);
+  --Message-padding-right: calc(var(--space-md) * 2);
+  --Message-background: var(--color-gray);
+  --Message-warning-background: var(--color-orange);
+  --Message-warning-color: var(--color-white);
+}
+
+.Message {
+  position: relative;
+  padding: var(--Message-padding);
+  padding-right: var(--Message-padding-right); /* room for close button if present */
+  background: var(--Message-background);
+}
+
+.Message-close {
+  position: absolute;
+  display: block;
+  top: var(--Message-padding);
+  right: var(--Message-padding);
```

Comment:
Per my other comment, positioning for this may be simpler if [you use Flexbox](http://codepen.io/tylersticka/pen/0d624f543bf0653c3e9cab12d525d7d5).